admiral/pkg/clusters/virtualservice_routing_test.go

```diff
@@ -171,6 +171,7 @@ func TestAddUpdateVirtualServicesForIngress(t *testing.T) {
 	testCases := []struct {
 		name                        string
```

Comment:
nitpick

i prefer to have a simple test case struct 
```go
struct {
  name string
  args args
  want want
}
```

This way it is pretty clear what the inputs and outputs are.
Not super important though.